UnitTests/UnitTests.vcxproj

```diff
@@ -88,6 +88,7 @@
       <PreprocessorDefinitions>_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
       <OpenMPSupport>true</OpenMPSupport>
+      <AdditionalIncludeDirectories>C:\Users\asus\source\repos\VladGribinchuk\FoundationBuilder\GeometryCore;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
Here is the absolute path. If you really need this additional include directory, then use only relative path

Reply:
It was created accidentally, i will try to remove it

Reply:
How to fix it? As i understood, this line is unnecessary here, because it is path in my pc. But how did it get here?) I was looking for in properties, but i didn't find anything